src/Core/Languages/Actions/CreateLanguage.php

```diff
@@ -26,8 +26,13 @@ public function authorize()
     public function rules()
     {
         return [
-            'lang' => 'required|string',
-            'iso' => 'required|string|unique:languages,iso',
+            'code' => [
+                'required',
+                'string',
+                'unique:languages,code',
+                // See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language
+                'regex:/^[a-zA-Z0-9 *,-.;=]*$/',
```

Comment:
Should be `'regex:/^[a-zA-Z0-9-]*$/',`

Reply:
I'm confused this is from the Mozilla page and something we discussed on the call?

yes, with the additional restriction that values can only be 0-9, A-Z, a-z, space or *,-.;=.